src/test/kotlin/no/nav/dagpenger/regel/api/grunnlag/GrunnlagRouteTest.kt

```diff
@@ -60,6 +63,16 @@ class GrunnlagRouteTest {
     fun `Subpaths subsumsjon and status are present`() {
         val storeMock = mockk<SubsumsjonStore>(relaxed = true)
 
+        val subsumsjon = mockk<GrunnlagSubsumsjon>(relaxed = true)
```

Comment:
I got build error `Could not compute caller for function: private constructor Subsumsjon()` - i do not think moshi are able to deserialize